Sources/MockingMacros/Extensions/InheritedTypeSyntax/InheritedTypeSyntax+UncheckedSendable.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  InheritedTypeSyntax+UncheckedSendable.swift
+//
+//  Copyright © 2025 Fetch.
+//
+
+import SwiftSyntax
+
+extension InheritedTypeSyntax {
+
+    /// An `InheritedTypeSyntax` representing unchecked sendable conformance
+    /// with a trailing comma.
+    ///
+    /// ```swift
+    /// // Represents
+    /// @unchecked Sendable,
+    /// ```
+    static let uncheckedSendable = InheritedTypeSyntax(
+        type: AttributedTypeSyntax(
+            specifiers: [],
+            attributes: AttributeListSyntax {
+                AttributeSyntax(
+                    attributeName: IdentifierTypeSyntax(
+                        name: "unchecked"
+                    )
+                )
+            },
+            baseType: IdentifierTypeSyntax(name: "Sendable")
+        ),
+        trailingComma: .commaToken()
+    )
```

Comment:
Even though you're specifying in the documentation that the trailing comma is included, it feels weird to me to include it by default. If anything, I would expect the default to be no trailing comma. I think this should be a constructor function that takes `trailingComma` as an argument instead:
```suggestion
    /// Returns an `InheritedTypeSyntax` representing `@unchecked Sendable`
    /// conformance.
    ///
    /// - Parameter trailingComma: The trailing comma token to apply to the 
    ///   inherited type syntax, or `nil` to omit the trailing comma.
    /// - Returns: An `InheritedTypeSyntax` representing `@unchecked Sendable`
    ///   conformance.
    static func uncheckedSendable(
        trailingComma: TokenSyntax? = nil
    ) -> InheritedTypeSyntax {
        InheritedTypeSyntax(
            type: AttributedTypeSyntax(
                specifiers: [],
                attributes: AttributeListSyntax {
                    AttributeSyntax(
                        attributeName: IdentifierTypeSyntax(
                            name: "unchecked"
                        )
                    )
                },
                baseType: IdentifierTypeSyntax(name: "Sendable")
            ),
            trailingComma: trailingComma
        )
    }
```

Reply:
Alternatively, you keep it the same but remove the trailing comma, and then at the calcite, you just use `.with(\.trailingComma, .commaToken)`. That also seems like a reasonable option to me.
```suggestion
    /// An `InheritedTypeSyntax` representing `@unchecked Sendable` conformance.
    ///
    /// ```swift
    /// @unchecked Sendable
    /// ```
    static let uncheckedSendable = InheritedTypeSyntax(
        type: AttributedTypeSyntax(
            specifiers: [],
            attributes: AttributeListSyntax {
                AttributeSyntax(
                    attributeName: IdentifierTypeSyntax(
                        name: "unchecked"
                    )
                )
            },
            baseType: IdentifierTypeSyntax(name: "Sendable")
        )
    )
```